packages/bits/src/services/unit-conversion.service.ts

```diff
@@ -55,6 +60,7 @@ export class UnitConversionService {
         return {
             value: strValue,
             order: resultOrder,
+            scientificNotation: value?.toExponential(scale).toLocaleString(),
```

Comment:
I don't think String.prototype.toLocaleString actually does anything natively. It only works on numbers. May as well remove this.